packages/components/src/components/contract-card/contract-card.tsx

```diff
@@ -41,21 +41,22 @@ const ContractCard = ({
     toggleUnsupportedContractModal,
 }: React.PropsWithChildren<TContractCardProps>) => {
     const fallback_result = profit_loss >= 0 ? 'won' : 'lost';
-    const payout = formatMoney(contract_info.currency ?? '', profit_loss > 0 ? profit_loss : 0, true);
+    const payout_info = is_multiplier ? getTotalProfit(contract_info) : profit_loss;
```

Comment:
As I checked, value from `getTotalProfit(contract_info)` is applicable only for `Multipliers` ( as it included comission), but for `Vanillas`, `Accumulators`, `Turbos` it's showing value different from `Total profit/loss`. So, for all the rest contracts I left old good `profit_loss`